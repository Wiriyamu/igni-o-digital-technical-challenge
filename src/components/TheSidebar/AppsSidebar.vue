<template>
  <nav class="nav">
    <p class="nav-apps">Apps</p>
    <ul>
      <li class="nav-item">
        <img src="@/assets/images/TheSidebar/mail.svg" alt="Email" />
        <p class="nav-item-text">Email</p>
      </li>
      <li class="nav-item">
        <img
          src="@/assets/images/TheSidebar/message-square.svg"
          alt="Conversar"
        />
        <p class="nav-item-text">Conversar</p>
      </li>
      <li class="nav-item">
        <img src="@/assets/images/TheSidebar/check-square.svg" alt="Todo" />
        <p class="nav-item-text">Todo</p>
      </li>
      <li class="nav-item">
        <img src="@/assets/images/TheSidebar/calendar.svg" alt="Calendário" />
        <p class="nav-item-text">Calendário</p>
      </li>
    </ul>
  </nav>
</template>

<script>
export default {
  name: 'AppsSidebar'
}
</script>

<style lang="scss" scoped>
.nav {
  margin: 50px 0 10px 30px;
}

.nav-apps {
  margin-bottom: 10px;
  font-weight: 300;
  text-transform: uppercase;
  color: #999999;
}

.nav-item {
  display: flex;
  margin: 20px 0;
  cursor: pointer;

  &:hover {
    opacity: 0.8;
  }
}

.nav-item-text {
  margin-left: 10px;
  color: #2c2c2c;
}
</style>
