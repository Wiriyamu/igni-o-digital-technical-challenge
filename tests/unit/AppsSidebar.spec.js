import AppsSidebar from '@/components/TheSidebar/AppsSidebar'
import { mount } from '@vue/test-utils'

describe('AppsSidebar.vue', () => {
  it('Snapshot component', () => {
    const wrapper = mount(AppsSidebar)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
