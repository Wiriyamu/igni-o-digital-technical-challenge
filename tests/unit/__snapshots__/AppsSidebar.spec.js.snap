// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppsSidebar.vue Snapshot component 1`] = `
<nav class="apps">
  <p class="apps__title">Apps</p>
  <ul class="apps__list">
    <li class="apps__item"><img src="" alt="Email">
      <p class="apps__item-text">Email</p>
    </li>
    <li class="apps__item"><img src="" alt="Conversar">
      <p class="apps__item-text">Conversar</p>
    </li>
    <li class="apps__item"><img src="" alt="Todo">
      <p class="apps__item-text">Todo</p>
    </li>
    <li class="apps__item"><img src="" alt="Calendário">
      <p class="apps__item-text">Calendário</p>
    </li>
  </ul>
</nav>
`;
